benchmarks/viscoelastic_relaxation/viscoelastic_relaxation.prm

```diff
@@ -0,0 +1,185 @@
+# This setup models the relaxation of elastic stresses in the absence
+# of drivers.
+# The analytical solution is:
+#     \tau_{xx} = \tau_{xx_{t0}} \exp(-t \frac{\mu}{\eta_{viscous}}),
+# where \tau_{xx} is the first component of the stored stress tensor,
+# \mu is the shear modulus and \eta_{viscous} the viscous viscosity.
+#  Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 250e3
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = iterated Advection and Stokes
+set Nonlinear solver tolerance             = 1e-6
+set Max nonlinear iterations               = 20
+set CFL number                             = 0.5
+set Maximum time step                      = 125
+set Pressure normalization                 = surface
+set Surface pressure                       = 0.
+set Output directory                       = ve_relaxation_dt125yr_dh10km
+
+# New elasticity settings
+set Use operator splitting                 = true
+
+# Solver settings
+subsection Solver parameters
+  set Temperature solver tolerance = 1e-5
+  subsection Stokes solver parameters
+    set Use direct solver for Stokes system = false
+    set Linear solver tolerance = 1e-7
+    set Number of cheap Stokes solver steps = 200
+  end
+  # Make sure to do only 1 splitting step
+  subsection Operator splitting parameters
+    set Reaction time step                 = 5000 # larger than maximum Stokes time step
+    set Reaction time steps per advection step = 1
+  end
+end
+
+# Model geometry (100x100 km, 2 km spacing)
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X repetitions = 10
+    set Y repetitions = 10
+    set X extent      = 100e3
+    set Y extent      = 100e3
+  end
+end
+
+# Mesh refinement specifications
+subsection Mesh refinement
+  set Initial adaptive refinement        = 0
+  set Initial global refinement          = 0
+  set Time steps between mesh refinement = 0
+end
+
+# Element types
+subsection Discretization
+  set Composition polynomial degree     = 2
+  set Stokes velocity polynomial degree = 2
+  set Temperature polynomial degree     = 1
```

Comment:
Indeed, temperature is irrelevant. However, I've now set the degree back to 2, because when using only 1 cell (so no repetitions, no refinement), the temperature is not set correctly, giving an error of 0 K in the material model, even though temperature is set to 293 K. 
```
An error occurred in line <165> of file </home/bbpanneg/software/aspect/aspect_fix_stresses_elasticity/aspect/source/material_model/rheology/visco_plastic.cc> in function
    aspect::MaterialModel::IsostrainViscosities aspect::MaterialModel::Rheology::ViscoPlastic<dim>::calculate_isostrain_viscosities(const aspect::MaterialModel::MaterialModelInputs<dim>&, unsigned int, const std::vector<double, std::allocator<double> >&, const std::vector<double, std::allocator<double> >&, const std::vector<unsigned int>&) const [with int dim = 2]
The violated condition was: 
    temperature_for_viscosity != 0
Additional information: 
    The temperature used in the calculation of the visco-plastic rheology
    is zero. This is not allowed, because this value is used to divide
    through. It is probably being caused by the temperature being zero
    somewhere in the model. The relevant values for debugging are:
    temperature (0), adiabatic_temperature_gradient_for_viscosity (0) and 
    pressure (0).
```